opal/mca/threads/argobots/threads_argobots.h.in

```diff
@@ -26,7 +26,7 @@
 #ifndef OPAL_MCA_THREADS_ARGOBOTS_THREADS_ARGOBOTS_H
 #define OPAL_MCA_THREADS_ARGOBOTS_THREADS_ARGOBOTS_H
 
-#include <abt.h>
+#include "@OPAL_ARGO_INCLUDE_PATH@abt.h"
```

Comment:
I can't find any other uses of this style in OMPI.  Is there a reason to do a config variable substitution instead of adding the path to `THREAD_CPPFLAGS`?  Does `abt.h` occur in other packages which could confuse configure?

Reply:
Yes this is different.  I did this to avoid having to make changes to Makefile.am's in many other places, and wanted to avoid forcing the abt.h include path into CFLAGS. etc.

Reply:
This sounds OK to me.